### PR TITLE
fixed some database structure

### PR DESCRIPTION
**Pull request by ShaileshS1702**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-This is a [Next.js](https://nextjs.org/) project bootstrapped with [`create-next-app`](https://github.com/vercel/next.js/tree/canary/packages/create-next-app).
+yarThis is a [Next.js](https://nextjs.org/) project bootstrapped with [`create-next-app`](https://github.com/vercel/next.js/tree/canary/packages/create-next-app).
```
Comment (Bestem0r) [inline README.md]: Typo

File: scarpe.js [hunk 2]
```diff
@@ -0,0 +1,10 @@
+
+
+//lagre = document.getElementById("lagre")
```
Comment (Bestem0r) [inline scarpe.js]: Remove test comment

File: src/app/api/mongoose.ts [hunk 3]
```diff
@@ -0,0 +1,7 @@
+import mongoose from "mongoose";
+const uri = "mongodb://127.0.0.1/pizza";
+
+
+const connection =  mongoose.connect(uri); //kobler til hele mongo-systemet
```
Comment (Bestem0r) [inline src/app/api/mongoose.ts]: Document in English

File: src/app/api/pizza/route.ts [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+
+
+
+
+export async function GET(request: NextRequest) {
+
+  await connection;
+  await Pizza.init();
+  const pizzas = await Pizza.find({});
+  console.log(pizzas);
```
Comment (Bestem0r) [inline src/app/api/pizza/route.ts]: Remove console log

File: src/app/login/page2.tsx [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+import { KeyboardEvent, useEffect } from 'react'
+// etc...
+
+const Component = () => {
+
+  useEffect(() => {
+    const keyDownHandler = (e:any) => console.log(`You pressed ${e.code}.`);
+    document.addEventListener("keydown", keyDownHandler);
+
+    // clean up
```
Comment (Bestem0r) [inline src/app/login/page2.tsx]: Maybe remove this comment as well?

**Review by Bestem0r — CHANGES_REQUESTED** [review]
Very nice! Some nitpicks

File: src/app/api/users/route.ts [hunk 6]
```diff
@@ -0,0 +1,16 @@
+import { NextRequest, NextResponse } from "next/server";
+import connection from "../mongoose";
+import User from "../models/users";
+
+
+
+
```
Comment (magnusbrecke) [inline src/app/api/users/route.ts]: Delete the spaces for nicer format

**Review by magnusbrecke — APPROVED** [review]
LGTM!

**Review by Bestem0r — APPROVED** [review]
Very nice!

**Pull request merged**